wrapanapi/msazure.py

```diff
@@ -478,8 +489,13 @@ def copy_blob_image(self, template, vm_name, storage_account,
         return operation.status
 
     def list_template(self):
-        self.logger.info("Attempting to List Azure VHDs in templates directory")
-        return self.list_blob_images(container=self.template_container)
+        """
+        Returns a list of VHDs/Images which might be used as provision template for Instance
+        """
+        self.logger.info("Attempting to List Azure VHDs/Images")
+        return self.list_blob_images(container=self.template_container)\
```

Comment:
Can use `list_blob_images().extend()` here too.

Reply:
I thought here it would be too many nested calls. and selected readability